phoenix-pherf/src/main/java/org/apache/phoenix/pherf/configuration/QuerySet.java

```diff
@@ -31,7 +31,24 @@
 	private long numberOfExecutions = PherfConstants.DEFAULT_NUMBER_OF_EXECUTIONS;
 	private long executionDurationInMs = PherfConstants.DEFAULT_THREAD_DURATION_IN_MS;
 	private ExecutionType executionType = ExecutionType.SERIAL;
-
+	private boolean randomPointRead = false;
+	private String primaryKey = PherfConstants.DEFAULT_PRIMARY_KEY;
+	
+	@XmlAttribute
+	public String getPrimaryKey() {
+		return primaryKey;
```

Comment:
nit: more formatting. Make sure you import the code stye preferences into your project.